### ISSUE TITLE
Need Help Train not running success?

### ISSUE THREAD

**Issue by MartinH04** [zejunwang1/bert_text_classification#1]
When I run python main.py --mode train --data_dir ./data --pretrained_bert_dir ./pretrained_bert
loading data...
202752it [00:00, 1377265.38it/s]
18980it [00:00, 1325163.80it/s]
time usage: 0:00:00
It's train mode
Epoch [1/3]
Epoch [2/3]
Epoch [3/3]
then the shell exit，not print the line
Iter:      0, Batch Train Loss:   2.4, Batch Train Acc: 13.28%, Val Loss:   2.3, Val Acc: 13.82%, Time: 0:00:19 *
Iter:    100, Batch Train Loss:  0.31, Batch Train Acc: 90.62%, Val Loss:  0.35, Val Acc: 89.44%, Time: 0:01:42 *
Looking forward to your reply，Thanks

**Comment by MartinH04** [comment]
It's working,The data txt use tab to split the word and nums
